src/test/java/teammates/test/cases/ui/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -0,0 +1,173 @@
+        
+        CourseAttributes course6 = dataBundle.courses.get("course6");
+        params = new String[]{
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, "First Session",
+                Const.ParamsNames.COURSE_ID, course.id,
+                Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME, "First Session",
+                Const.ParamsNames.COPIED_COURSES_ID, course.id,
+                Const.ParamsNames.COPIED_COURSES_ID, course6.id
+        };
+        
+        a = getAction(params);
+        rr = (RedirectResult) a.executeAndPostProcess();
+        
+        assertEquals(Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE
+                        + "?error=true"
+                        + "&user="
+                        + instructor.googleId
+                        + "&courseid="
+                        + instructor.courseId
+                        + "&fsname=First+Session",
+                rr.getDestinationWithParams());
+        
+        assertEquals("A feedback session with the name \"First Session\" already exists in the following course(s): FeedbackEditCopy.CS2104.", rr.getStatusMessage());
+        
+        
+        ______TS("Failure case: empty name");
+        
+        params = new String[]{
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, "First Session",
+                Const.ParamsNames.COURSE_ID, course.id,
+                Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME, "",
+                Const.ParamsNames.COPIED_COURSES_ID, course.id,
+                Const.ParamsNames.COPIED_COURSES_ID, course6.id
+        };
+        
+        a = getAction(params);
+        rr = (RedirectResult) a.executeAndPostProcess();
+        
+        assertEquals(Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE
+                        + "?error=true"
+                        + "&user="
+                        + instructor.googleId
+                        + "&courseid="
+                        + instructor.courseId
+                        + "&fsname=First+Session",
+                rr.getDestinationWithParams());
+        
+        assertEquals("\"\" is not acceptable to TEAMMATES as feedback session name because it is empty. The value of feedback session name should be no longer than 38 characters. It should not be empty.", rr.getStatusMessage());
+        
+        String expectedLogMessage =
+                "TEAMMATESLOG|||instructorFeedbackEditCopy|||instructorFeedbackEditCopy|||true|||Instructor|||Instructor 2|||" + 
+                "FeedbackEditCopyinstructor2|||tmms.instr@gmail.tmt|||Servlet Action Failure : \"\" is not acceptable to TEAMMATES " +
+                "as feedback session name because it is empty. The value of feedback session name should be no longer than 38 characters. " +
+                "It should not be empty.|||/page/instructorFeedbackEditCopy";
+        assertEquals(expectedLogMessage, a.getLogMessage());
+        
```

Comment:
Is there a test case for invalid/non-existing feedback session name/course?
Need to test for access controls as well.